tools/bin/gke-kube-acceptance-test/acceptance_test_kube_gke.sh

```diff
@@ -44,7 +44,7 @@ function findAndDeleteTag () {
 }
 
 function cleanUpImages () {
-    TOKEN=$(curl --request POST 'https://hub.docker.com/v2/users/login/' --header 'Content-Type: application/json' --data-raw '{"username":"airbytebot","password":"'$DOCKER_PASSWORD'"}' | jq '.token')
+    TOKEN=$(curl --request POST 'https://hub.docker.com/v2/users/login/' --header 'Content-Type: application/json' --data-raw '{"username":"'$DOCKER_HUB_USERNAME'","password":"'$DOCKER_HUB_PASSWORD'"}' | jq '.token')
```

Comment:
hmm - this one is also grabbing a token through curl, do we need to `set +x` `set -x` here as well?

Reply:
I don't think so, because no where in the file do we turn that on (set +x)?

Reply:
... but maybe we should?

Reply:
Ah! I see, it was specifically enabled on `manage.sh` - should be ok then